api/src/main/java/org/openmrs/module/hospitalcore/db/InventoryCommonDAO.java

```diff
@@ -55,4 +55,7 @@ public interface InventoryCommonDAO {
     Regimen createRegimen(Regimen regimen);
     Regimen updateRegimen(Regimen regimen);
     void voidRegimen(Regimen regimen);
+
```

Comment:
We also need a voidCycle method
```suggestion
void voidCycle(Cycle cycle) throws APIException;
```